x-pack/agent/pkg/agent/program/testdata/single_config-filebeat.yml

```diff
@@ -4,10 +4,13 @@ filebeat:
     paths:
       - /var/log/hello1.log
       - /var/log/hello2.log
+    index: filebeat-generic-default
```

Comment:
This is incorrect, it should be: 
```
logs-{dataset}-{namespace}
```

@ruflin When no dataset and namespace are defined what are the default values for them? The https://github.com/elastic/kibana/pull/56132 doesn't mention it?

Reply:
https://github.com/elastic/kibana/pull/56132/files#diff-abbe71b916f44ea720e680dd923aba1cR101

Reply:
this issue mentions defaults #15691
for NS it is `default` for dataset `generic` so this follows `logs-dataset-namespace` correctly

Reply:
The above should be `logs-generic-default`.

Reply:
I believe I've read it too fast. thanks